packages/esm-patient-medications-app/src/drug-order-basket-panel/drug-order-basket-panel.extension.tsx

```diff
@@ -58,13 +58,15 @@ export default function DrugOrderBasketPanelExtension() {
     closeWorkspace('order-basket', {
       ignoreChanges: true,
       onWorkspaceClose: () => launchPatientWorkspace('add-drug-order'),
+      closeWorkspaceGroup: false,
     });
   };
 
   const openDrugForm = (order: DrugOrderBasketItem) => {
     closeWorkspace('order-basket', {
       ignoreChanges: true,
       onWorkspaceClose: () => launchPatientWorkspace('add-drug-order', { order }),
+      closeWorkspaceGroup: false,
```

Comment:
Shouldn't this be the default, assuming that there's at least one workspace in this group that hasn't been closed?

Reply:
> that there's at least one workspace in this group that hasn't been closed?

Actually no, since the new workspace is launched only after the previous workspace closes. For now, it's not possible to know whether a new workspace will be opened after closing the current one or not, hence we have the `closeWorkspaceGroup` property set to `true` by default to clear the workspace group and run cleanup. In case we don't want to clear the workspace group, similar to the above use case, we can pass the `closeWorkspaceGroup` to false.
Refer: https://github.com/openmrs/openmrs-esm-core/blob/78b7975f08b975e6080c5224bb745bd8be97beba/packages/framework/esm-styleguide/src/workspaces/workspaces.ts#L249